### ISSUE TITLE
Looks  weird on HiDPI display

### ISSUE THREAD

**Issue by anandtrex** [jake-phy/WindowIconList#87]
See attached image. WindowIconList looks really weird on a HiDPI display (2560x1440 14") compared to the normal window list shown next to it for comparison.

![windowiconlist-hidpi-screenshot](https://cloud.githubusercontent.com/assets/1858015/6838320/9ba8c8f0-d357-11e4-9f05-ef03f54f7db8.png)

**Comment by jake-phy** [comment]
hmm that is weird.  Not sure why the icons are squished.

**Comment by xcjs** [comment]
I don't know if this is the correct answer (I've never done Cinnamon applet development) but I was aware that GTK 3 extensions were written in JavaScript so I decided to investigate.

While I don't understand the underlying cause, this is triggered by not explicitly setting the icon width. I took a look at specialButtons.js and added the following code after line 60:

    this.actor.height = parent._applet._panelHeight - 2;
    this.actor.width = parent._applet._panelHeight * 1.25;

The icons are roughly (very roughly since I only did it by sight) 1.25 times wider than they are tall. After modifying the extension as mentioned, things seem to work out pretty well.

Should I make a pull request?

**Comment by jake-phy** [comment]
yeah go ahead and I will take a look.  sorry it took 5 months to reply :(.

**Comment by grenzor** [comment]
Also facing this issue with HiDPI. Possible you can take @xcjs solution up?

**Comment by jake-phy** [comment]
Ok can you try now?  Make sure you use the develop branch.

**Comment by grenzor** [comment]
Using the dev branch, the icons are still appearing as large
![x](https://cloud.githubusercontent.com/assets/13731217/12416404/7556454e-bef5-11e5-905e-1a304e7371f4.png)
